catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogPropertiesMetadata.java

```diff
@@ -53,6 +53,10 @@ public class HadoopCatalogPropertiesMetadata extends BaseCatalogPropertiesMetada
    */
   public static final String DEFAULT_FS_PROVIDER = "default-filesystem-provider";
 
+  static final String FILESYSTEM_CONNECTION_TIMEOUT_SECONDS =
+      "filesystem-connection-timeout-seconds";
+  static final int DEFAULT_GET_FILESYSTEM_TIMEOUT_SECONDS = 6;
```

Comment:
Why it is the "6" seconds we defined here?

Reply:
This is just an experience value. This specific setting depends on the environment at that time. I believe 6s is long enough to get the connection.